test/experiment/training.jl

```diff
@@ -95,78 +103,88 @@ end
-                        nodeChain = Flux.Chain(
-                            Flux.Dense(20, 20),
-                        ),
-                        outputLayer = Flux.Dense(20, generator.nb_nodes)
-                    ),
-                    optimizer = ADAM(0.001f0)
+                    model = approximator_model,
+                    optimizer = ADAM(0.0005f0)
                 ),
                 target_approximator = RL.NeuralNetworkApproximator(
-                    model = SeaPearl.FlexGNN(
-                        graphChain = Flux.Chain(
-                            GeometricFlux.GCNConv(numInFeatures => 20),
-                            GeometricFlux.GCNConv(20 => 20),
-                        ),
-                        nodeChain = Flux.Chain(
-                            Flux.Dense(20, 20),
-                        ),
-                        outputLayer = Flux.Dense(20, generator.nb_nodes)
-                    ),
-                    optimizer = ADAM(0.001f0)
+                    model = target_approximator_model,
+                    optimizer = ADAM(0.0005f0)
                 ),
                 loss_func = Flux.Losses.huber_loss,
                 stack_size = nothing,
                 γ = 0.99f0,
-                batch_size = 2,
-                update_horizon = 1,
-                min_replay_history = 1,
-                update_freq = 1,
-                target_update_freq = 100,
+                batch_size = 32,
+                update_horizon = 4,
+                min_replay_history = 32,
+                update_freq = 4,
+                target_update_freq = 20,
             ), 
             explorer = RL.EpsilonGreedyExplorer(
-                ϵ_stable = 0.01,
+                ϵ_stable = 0.001,
                 kind = :exp,
                 ϵ_init = 1.0,
                 warmup_steps = 0,
-                decay_steps = 500,
+                decay_steps = 5000,
                 step = 1,
                 is_break_tie = false, 
                 #is_training = true,
                 rng = MersenneTwister(33)
             )
         ),
         trajectory = RL.CircularArraySLARTTrajectory(
-            capacity = 500,
-            state = Matrix{Float32} => state_size,
+            capacity = 800,
+            state = SeaPearl.DefaultTrajectoryState[] => (),
             legal_actions_mask = Vector{Bool} => (generator.nb_nodes, ),
+
         )
     )
 
-    learnedHeuristic = SeaPearl.LearnedHeuristic(agent, maxNumberOfCPNodes)
-
+    learnedHeuristic = SeaPearl.LearnedHeuristic(agent)
     initial_params = deepcopy(params(learnedHeuristic.agent.policy.learner.approximator.model))
 
     metrics,evalmetrics = SeaPearl.train!(
         valueSelectionArray=learnedHeuristic, 
         generator=generator,
-        nbEpisodes=3,
+        nbEpisodes=10,
         strategy=SeaPearl.DFSearch,
-        variableHeuristic=SeaPearl.MinDomainVariableSelection{false}(),
+        variableHeuristic=SeaPearl.MinDomainVariableSelection{true}(),
```

Comment:
objective function ( number of used color ) was not used by the variableHeuristic.